wemake_python_styleguide/logic/tree/imports.py

```diff
@@ -1,7 +1,9 @@
 from typing import List
 
 from wemake_python_styleguide import constants
-from wemake_python_styleguide.logic.naming import logical
+from wemake_python_styleguide.logic.naming.logical import (  # noqa: I001
```

Comment:
Sorry, `I001` is not allowed to be ignored. This gave me an idea for a new rule! Thank you! https://github.com/wemake-services/wemake-python-styleguide/issues/1248

Reply:
I had strange problems with order of imports. However, `isort` could help me

Reply:
What kind of problems? Do you want to report a new issue?